TestStack.BDDfy/Reporters/Html/ClassicReportBuilder.cs

```diff
@@ -148,7 +148,7 @@ private void Footer()
             EmbedJavascriptFile(_model.CustomJavascript, HtmlReportResources.CustomJavascriptComment);
         }
 
-        private void AddStory(Story story)
+        private void AddStory(ReportModel.Story story)
```

Comment:
To avoid confusion, do you think it's worth calling these StoryModel, ScenarioModel etc?